wasm/src/algorithms/poseidon/mod.rs

```diff
@@ -27,14 +27,17 @@ pub use poseidon8::Poseidon8;
 mod tests {
     use super::*;
     use crate::{
-        Field,
-        Group,
-        Scalar,
-        types::native::{FieldNative, Poseidon2Native, Poseidon4Native, Poseidon8Native},
-        utilities::test::{create_native_field_vector, js_array_from_fields},
+        js_array_from_fields, native::{GroupNative, LiteralNative, PlaintextNative, ScalarNative}, test::TEST_STRUCT, types::native::{FieldNative, Poseidon2Native, Poseidon4Native, Poseidon8Native}, utilities::test::create_native_field_vector, Field, Group, Scalar
+    };
+    use snarkvm_console::{
+        algorithms::{Hash, HashMany, HashToGroup, HashToScalar},
+        prelude::ToFields,
+        program::{Literal, Plaintext},
```

Comment:
These are the same types as LiteralNative and PlaintextNative respectively. So they shouldn't be directly imported. Any imports of a type we export like `Plaintext` should import the `wasm` struct we define for it.

```suggestion
```